app/services/redirection_creation.rb

```diff
@@ -1,15 +1,16 @@
 class RedirectionCreation
-  def self.perform(referrer, slug)
-    new(referrer, slug).perform
+  def self.perform(referrer, slug, closed:)
+    new(referrer, slug, closed: closed).perform
```

Comment:
With newer ruby versions (3.1 I think?), you can just use:

```suggestion
    new(referrer, slug, closed:).perform
```

I care about a 1/5 about this, but thought you should know.